posts/cuda-aware-mpi-example/src/Makefile

```diff
@@ -35,13 +35,15 @@ MPICFLAGS=-I${MPI_HOME}/include
 CUDACFLAGS=-I${CUDA_INSTALL_PATH}/include
 
 GENCODE_SM13    := -gencode arch=compute_13,code=sm_13
-GENCODE_SM20    := -gencode arch=compute_20,code=sm_20
 GENCODE_SM30    := -gencode arch=compute_30,code=sm_30
 GENCODE_SM35    := -gencode arch=compute_35,code=sm_35
 GENCODE_SM35    := -gencode arch=compute_37,code=sm_37
 GENCODE_SM50    := -gencode arch=compute_50,code=sm_50
-GENCODE_SM52    := -gencode arch=compute_52,code=\"sm_52,compute_52\"
-GENCODE_FLAGS   := $(GENCODE_SM20) $(GENCODE_SM30) $(GENCODE_SM35) $(GENCODE_SM37) $(GENCODE_SM50) $(GENCODE_SM52)
+GENCODE_SM52    := -gencode arch=compute_52,code=sm_52
+GENCODE_SM60    := -gencode arch=compute_60,code=sm_60
+GENCODE_SM70    := -gencode arch=compute_70,code=\"sm_70,compute_70\"
+GENCODE_FLAGS   := $(GENCODE_SM20) $(GENCODE_SM30) $(GENCODE_SM35)             \
```

Comment:
Please remove GENCODE_SM20 here since it is removed above. (Note you don't have to open a separate PR with each change. Just push changes to your branch.)

Reply:
Oh My bad, missed to remove that line. Will post a rev.

Thanks